src/Debugging/Drivers/ErrorLogDebugger.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Saloon\Debugging\Drivers;
+
+use Saloon\Debugging\DebugData;
+
+class ErrorLogDebugger implements DebuggingDriver
+{
+    public function name(): string
+    {
+        return 'error_log';
+    }
+
+    /**
+     * @param \Saloon\Debugging\DebugData $data
+     *
+     * @return $this
+     */
+    public function send(DebugData $data): static
+    {
+        error_log(print_r($this->formatData($data), true), LOG_DEBUG);
+
+        return $this;
+    }
+
+    /**
+     * @return array<string, mixed>
+     */
+    protected function formatData(DebugData $data): array
+    {
+        // TODO: Format the $data, and send it appropriately to the error_log().
+
+        $formattedData = [
+            'method' => $data->method(),
+            'uri' => $data->url(),
+            'request_headers' => $data->pendingRequest()->headers(),
+            'request_query' => $data->pendingRequest()->query()->all(),
+            'request_payload' => $data->pendingRequest()->body()?->all(),
+        ];
+
+        if ($data->wasSent()) {
+            $formattedData += [
+                'response_status' => $data->response()->status(),
+                'response_headers' => $data->response()->headers(),
+
+                // TODO: This should be converted to body/content, and stuff like that.
+                'response_body' => $data->response()->json(),
```

Comment:
I think we should just use `->body()` for the raw string body, maybe we could check based on Content-Type too for some light formatting, JSON formatting only perhaps?

Reply:
I've updated this to first check the Content-Type header, and, depending on that return `response()->json()`, and `response()->xml()`, respectively.
Or, as a default, 'force-call' `response()->json()`, and upon Exception, call `response()->xml()`.
If `response()->xml()` returns false (fail), we'll fallback to raw string.

The reason I try to make an array is that tools like Ray, or even print_r(), that I use for syslog/error log, all present data neater.

Although.. we now have an issue - xml() will return a SimpleXMLElement, so that might be an issue - we'll need to test that.